### DISCUSSION TITLE
Why do the number of checks for the weekly scorecard scans differ for projects?

### DISCUSSION THREAD

**Discussion by dynamic-entropy** [ossf/scorecard#3987] [Q&A] [answered]
Hello awesome people

I see a couple of missing checks in the results of CNCF projects, namely `CI-Tests`, `Contributors` and `Dependency Update Tool`.
https://securityscorecards.dev/viewer/?uri=github.com/kubernetes/kubernetes
(I see the same 15 checks across all CNCF projects). 

Whereas we know that the scorecards test suite tests for more for other repositories (also on github), for e.g. https://securityscorecards.dev/viewer/?uri=github.com/ossf/scorecard

Why do we have this difference? Do we have a configuration file for all the 1 million scanned repositories?

Cheers

**Comment by diogoteles08** [comment, marked as answer]
Hi @dynamic-entropy!

> Why do we have this difference? 

So, the main point is that github.com/ossf/scorecard is running the Scorecard GitHub Actions(see [the workflow file](https://github.com/ossf/scorecard/blob/main/.github/workflows/scorecard-analysis.yml)) and publishing the results, while Kubernetes (and I assume the other CNCF projects) is not. 

When a project doesn't run Scorecard themselves and publish the results, we use our own infrastructure to gather some results and make them available on our viewer. The implementation of the cronjob is available on [this folder](https://github.com/ossf/scorecard/tree/main/cron). The problem is that some of the checks (e.g. the ones you mentioned) are too API heavy and we currently can't use our infrastructure to gather them for all the repositories.

> Do we have a configuration file for all the 1 million scanned repositories?

Not sure I understood your question, but we do have a file listing all the repositories we evaluate in our cronjob. Check [this file](https://github.com/ossf/scorecard/blob/main/cron/internal/data/projects.csv) for github projects and [this one](https://github.com/ossf/scorecard/blob/main/cron/internal/data/gitlab-projects.csv) for gitlab.

**Comment by dynamic-entropy** [comment]
Hi @diogoteles08 
Thank you for the response and all the links.

>Not sure I understood your question, but we do have a file listing all the repositories we evaluate in our cronjob. Check [this file](https://github.com/ossf/scorecard/blob/main/cron/internal/data/projects.csv) for github projects and [this one](https://github.com/ossf/scorecard/blob/main/cron/internal/data/gitlab-projects.csv) for gitlab.

I meant if there is a configuration file that specifies which checks to skip (or run) for certain repositories.
You answered my question, it is the same for all repositories that do not run the scorecard themselves.
https://github.com/ossf/scorecard/blob/d58bfb03aab496807d8489e09da9883928465f59/cron/config/config.yaml#L48

Cheers
